packages/orbit-components/src/Textarea/__tests__/index.test.tsx

```diff
@@ -54,20 +51,23 @@ describe("Textarea", () => {
     expect(textarea).not.toBeInvalid();
     expect(textarea.parentElement).toHaveStyle({ marginBottom: "12px" });
     expect(textarea).toHaveStyle({ padding: "12px" });
-
-    await act(() => user.type(textarea, "kek"));
-    expect(onChange).toHaveBeenCalled();
   });
 
-  it("should have focus and blur", async () => {
```

Comment:
out of scope, but this is a prime example of a redundant test 😅 normally we'd pass native props to the `textarea` element, and we really don't need to test whether `textarea` works as a `textarea`. we'd maybe have one user-like test to check that you can type stuff in there and that it reacts to changes, that's it

Reply:
Yes… kinda. Because our `Textarea` component does not simply render a `<textarea>`. It renders a label, a span, a textarea (through `styled.textarea`) and possibly even a tooltip with more stuff inside.
I fully understand what you mean, but I think we are just asserting that callbacks are not lost in the middle of the inner logic of the component and that they are passed to the correct element.
So I don't think the problem is the test, but the implementation that makes the test reasonably justifiable. But, as you said, that is (even more) out of scope of this PR. Let's address this soon™ 🤞